TokenLib/README.md

```diff
@@ -269,7 +269,7 @@ var output = JSON.parse(solc.compileStandardWrapper(JSON.stringify(input)));
 Solc-js also provides a linking method if you have compiled binary code already with the placeholder. To link this library the call would be:
 
  ```js
- bytecode = solc.linkBytecode(bytecode, { 'TokenLib': '0x71ecde7c4b184558e8dba60d9f323d7a87411946' });
```

Comment:
Note that this line had a different address, [also `v1.0.0` but compiled with an older version of `solc`](https://etherscan.io/address/0x71ecde7c4b184558e8dba60d9f323d7a87411946#code). I'm not sure if this is related to `solc-js` in particular.

Reply:
No, just leftover from updating to the new library